tests/timeout_test.go

```diff
@@ -136,6 +141,27 @@ func testEvidenceTimeout(t *testing.T) {
 	assert.ErrorContains(t, err, "context deadline exceeded")
 }
 
+func testMetadataTimeout(t *testing.T) {
+	// Create mock server
+	url, cleanup := createSleepyRequestServer()
+	defer cleanup()
+
+	// Create services manager configuring to work with the mock server
+	details := metadataAuth.NewMetadataDetails()
+	details.SetUrl(url)
+	servicesManager, err := metadata.NewManager(createServiceConfigWithTimeout(t, details))
+	assert.NoError(t, err)
+
+	query := []byte("query body")
+	serviceDetails := metadataServices.QueryDetails{
+		Body: query,
+	}
+
+	// Expect timeout
+	_, err = servicesManager.GraphiqlQuery(serviceDetails)
```

Comment:
GraphiqlQuery ? not GraphqlQuery?
same for the name of the file graphiql.go

Reply:
done